src/models/cloudProviderIdentifier.ts

```diff
@@ -13,13 +13,26 @@ export enum CloudAdapterIdentifier {
 }
 
 export enum CloudProviderIdentifier {
-    GENEZIO = "genezio",
+    CAPYBARA_LINUX = "genezioCloud",
+    CAPYBARA = "genezioCloudUnikernel",
+    CLUSTER = "genezioCluster",
+    GENEZIO = "genezioAws",
```

Comment:
In my opinion, we should stop using camel case for this, it's very specific to JavaScript and does not look good.

```suggestion
    CAPYBARA_LINUX = "genezio-cloud",
    CAPYBARA = "genezio-unikernel",
    CLUSTER = "genezio-cluster",
    GENEZIO = "genezio-aws",
```